ee/codegen/src/types/vellum.ts

```diff
@@ -826,6 +826,11 @@ export type WorkflowTrigger =
       sourceHandleId: string;
     };
```

Comment:
DevinAI separate all three of these into their own types, one of which called `IntegrationTrigger`, then combine into the union below. We should likely have a `BaseTrigger` that the other three types extend from